### PR TITLE
Added Cassandra credentials dao

### PR DESCRIPTION
**Pull request by arturjoshi**

### CODE REVIEW

File: server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/model/Credentials.java [hunk 1]
```diff
@@ -22,6 +22,5 @@
 public interface Credentials extends ToDto<CredentialsDto> {
 
     String getId();
-    byte[] getCredentialsBody();
```
Comment (bkhablenko) [inline server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/model/Credentials.java]: Why did you remove this property?

File: server/common/nosql/cassandra-dao/src/main/java/org/kaaproject/kaa/server/common/nosql/cassandra/dao/model/CassandraCredentials.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    public String getApplicationId() {
+        return applicationId;
+    }
+
+    public void setApplicationId(String applicationId) {
+        this.applicationId = applicationId;
+    }
+
+    public ByteBuffer getCredentialsBody() {
+        return credentialsBody;
```
Comment (bkhablenko) [inline server/common/nosql/cassandra-dao/src/main/java/org/kaaproject/kaa/server/common/nosql/cassandra/dao/model/CassandraCredentials.java]: You should return the byte[] representation of the credentials body to conform to the Credentials interface.

File: server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/model/Credentials.java (hunk 1 above)
Comment (arturjoshi) [inline server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/model/Credentials.java]: Because CassandraCredentials class contains ByteBuffer credentialsBody field to interact with CassandraDB and hasn't byte[] getters and setters

**Comment by bkhablenko** [comment]
The pull request has been merged manually.
